README.md

```diff
@@ -65,23 +65,43 @@ Details of services available need to be applied to the Umbraco web application'
         {
           "<serviceAlias>": {
             "DisplayName": "",
+            "AuthenticationMethod": "",
```

Comment:
We should set the different options of the enum here separated by vertical bars I think, just to be consistent with what we have for `true|false` below.  Think it's also useful to see at a glance what the options are.

```suggestion
            "AuthenticationMethod": "OAuth2AuthorizationCode|OAuth2ClientCredentials|OAuth1|ApiKey",
```